### PR TITLE
PHP-8.4: Document http_(get|clear)_last_response_headers() functions

### PR DESCRIPTION
**Pull request by Girgias**

### CODE REVIEW

File: reference/network/functions/http-get-last-response-headers.xml [hunk 1]
```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8"?>
+<refentry xml:id="function.http-get-last-response-headers" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+ <refnamediv>
+  <refname>http_get_last_response_headers</refname>
+  <refpurpose>Retrieve lastest HTTP response headers</refpurpose>
```
Comment (cmb69) [inline reference/network/functions/http-get-last-response-headers.xml:5]:
```suggestion
  <refpurpose>Retrieve last HTTP response headers</refpurpose>
```

File: reference/network/functions/http-get-last-response-headers.xml [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+   &example.outputs.similar;
+   <screen>
+<![CDATA[
+array(9) {
+  [0]=>
+  string(15) "HTTP/1.1 200 OK"
+  [1]=>
+  string(35) "Date: Sat, 12 Apr 2008 17:30:38 GMT"
+  [2]=>
+  string(29) "Server: Apache/2.2.3 (CentOS)"
```
Comment (cmb69) [inline reference/network/functions/http-get-last-response-headers.xml:61]: Wow!

**Review by cmb69 — APPROVED** [review]
Thank you! Found a typo; otherwise looks good.

File: reference/network/functions/http-get-last-response-headers.xml (hunk 2 above)
Comment (Girgias) [reply reference/network/functions/http-get-last-response-headers.xml:61]: I did copy this over from the [`$http_response_headers`](https://www.php.net/manual/en/reserved.variables.httpresponseheader.php) probably should check again

**Pull request merged**